src/utils/api.js

```diff
@@ -1,9 +1,23 @@
-import axios from "axios";
+import axios from 'axios';
 
-const newsSite = axios.create({ baseURL: 'https://news-project-8i4e.onrender.com/api'});
+const newsApi = axios.create({
+  baseURL: 'https://news-project-8i4e.onrender.com/api',
+});
 
-export const getArticles = () => {
-  return newsSite.get('./articles').then(({data}) => {
+export const getArticles = (article_id) => {
```

Comment:
I appreciate you have tried to make this function dynamic - it could be clearer if you just have two separate functions for separate concerns here - get articles and get a single article